apps/testingaccessibility/src/utils/get-coupon-label.ts

```diff
@@ -0,0 +1,14 @@
+export const getCouponLabel = (couponType: string) => {
+  switch (couponType) {
+    case 'special':
+      return 'Launch Discount'
+    case 'site':
+      return 'Sale'
+    case 'ppp':
+      return 'Regional Pricing'
+    case 'bulk':
+      return 'Team Discount'
+    default:
+      return 'Discount'
+  }
+}
```

Comment:
added a little ribbon that shows applied coupon label. also helps with avoiding layout shifts.